libs/blocks/card/card.js

```diff
@@ -32,7 +32,15 @@ const addWrapper = (el, section, cardType) => {
 
   if (prevGrid) return;
 
-  const upClass = getUpFromSectionMetadata(section);
+  let upClass = getUpFromSectionMetadata(section);
+  // Authored w/ a typed out number reference... 'two-up' vs. '2-up'
+  const list = ['two-up', 'three-up', 'four-up', 'five-up'];
+  const ixd = list.findIndex(i => i.includes(upClass));
+  if (ixd > -1) {
+    upClass = `${ixd+2}-up`;
```

Comment:
Given L44 `const up = upClass?.replace('-', '') || '3up';` shouldn't we set this without the  `-` to begin with?